From 307bd142feef8f42d3adcea354b72f4c076470e4 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Tue, 16 Feb 2010 00:59:24 +0100
Subject: [PATCH 37/62] alsa: use default output port names

instead of coming up with pointless aliases, reuse the already established
names, for second headphones, and second speakers.
---
 src/modules/alsa/alsa-mixer.c                      |    2 --
 .../mixer/paths/analog-output-desktop-speaker.conf |    3 ++-
 .../mixer/paths/analog-output-headphones-2.conf    |    1 +
 .../alsa/mixer/paths/analog-output-headphones.conf |    1 +
 .../alsa/mixer/paths/analog-output-speaker.conf    |    1 +
 5 files changed, 5 insertions(+), 3 deletions(-)

diff --git a/src/modules/alsa/alsa-mixer.c b/src/modules/alsa/alsa-mixer.c
index cb814af..93f2ed0 100644
--- a/src/modules/alsa/alsa-mixer.c
+++ b/src/modules/alsa/alsa-mixer.c
@@ -1783,9 +1783,7 @@ static int path_verify(pa_alsa_path *p) {
         { "analog-output-headphones",   N_("Analog Headphones") },
         { "analog-output-lfe-on-mono",  N_("Analog Output (LFE)") },
         { "analog-output-mono",         N_("Analog Mono Output") },
-        { "analog-output-headphones-2", N_("Analog Headphones 2") },
         { "analog-output-speaker",      N_("Analog Speakers") }
-        { "analog-output-desktop-speaker", N_("Analog Speakers 2") }
     };
 
     pa_alsa_element *e;
diff --git a/src/modules/alsa/mixer/paths/analog-output-desktop-speaker.conf b/src/modules/alsa/mixer/paths/analog-output-desktop-speaker.conf
index 9f18b25..dfdecf4 100644
--- a/src/modules/alsa/mixer/paths/analog-output-desktop-speaker.conf
+++ b/src/modules/alsa/mixer/paths/analog-output-desktop-speaker.conf
@@ -19,7 +19,8 @@
 ; See analog-output.conf.common for an explanation on the directives
 
 [General]
-priority = 100
+priority = 101
+name = analog-output-speaker
 
 [Element Hardware Master]
 switch = mute
diff --git a/src/modules/alsa/mixer/paths/analog-output-headphones-2.conf b/src/modules/alsa/mixer/paths/analog-output-headphones-2.conf
index 580c798..e47543f 100644
--- a/src/modules/alsa/mixer/paths/analog-output-headphones-2.conf
+++ b/src/modules/alsa/mixer/paths/analog-output-headphones-2.conf
@@ -20,6 +20,7 @@
 
 [General]
 priority = 89
+name = analog-output-headphones
 
 [Element Hardware Master]
 switch = mute
diff --git a/src/modules/alsa/mixer/paths/analog-output-headphones.conf b/src/modules/alsa/mixer/paths/analog-output-headphones.conf
index 79ebc7f..1d7bb0b 100644
--- a/src/modules/alsa/mixer/paths/analog-output-headphones.conf
+++ b/src/modules/alsa/mixer/paths/analog-output-headphones.conf
@@ -20,6 +20,7 @@
 
 [General]
 priority = 90
+name = analog-output-headphones
 
 [Element Hardware Master]
 switch = mute
diff --git a/src/modules/alsa/mixer/paths/analog-output-speaker.conf b/src/modules/alsa/mixer/paths/analog-output-speaker.conf
index b133a2e..c6916d6 100644
--- a/src/modules/alsa/mixer/paths/analog-output-speaker.conf
+++ b/src/modules/alsa/mixer/paths/analog-output-speaker.conf
@@ -20,6 +20,7 @@
 
 [General]
 priority = 100
+name = analog-output-speaker
 
 [Element Hardware Master]
 switch = mute
-- 
1.7.0

